### PR TITLE
template rm implemented

### PR DESCRIPTION
**Pull request by ChristianBoehm**
@bketelsen please have a look for my suggestion implementing template rm. I'm happy for discussion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,14 @@
...
+### Create initial configuration
 ```
-lxdev config -c
-lxdev config -t
+lxdev init
 ```
 
-These commands write `$HOME/.lxdev.yaml` and `$HOME/.lxdev/profiles/*.yaml`, which are configuration files and templates for your containers.
-### Create base LXC Profiles
+This command write `$HOME/.lxdev.yaml`, `$HOME/.lxdev/profiles/*.yaml` and `$HOME/.lxdev/templates/realations.yaml`,  which are configuration files, templates and container realation store for your containers.
```
Comment (bketelsen) [inline README.md]: typo: "relations"

File: cmd/create.go [hunk 2]
```diff
@@ -41,6 +41,15 @@ var createCmd = &cobra.Command{
 			log.Error("Unable to create container: " + err.Error())
 			os.Exit(1)
 		}
+
+		// Store the LXC Image -> Container relationship
+		log.Running("Storing image container relation ")
+		err = lxclient.SetContainerTemplateRealation(name, template, true)
```
Comment (bketelsen) [inline cmd/create.go]: the lxclient package should be self-contained and have only code that talks to the lxc/lxd services.  This function `SetContainerTemplateRelation` is part of lxdev, and should be in the cmd package in a new file.

File: cmd/remove.go [hunk 3]
```diff
@@ -34,6 +34,13 @@ var removeCmd = &cobra.Command{
 			log.Error("Error executing command: " + err.Error())
 			os.Exit(1)
 		}
+
+		err = lxclient.SetContainerTemplateRealation(name, "", false)
+		if err != nil {
```
Comment (bketelsen) [inline cmd/remove.go]: Same as above, move the `SetContainerTemplateRelation` function out of the lxclient package

**Comment by ChristianBoehm** [comment]
Thanks for your comments and review. I'll send a patch.

**Comment by bketelsen** [comment]
Thank you for the PR!

**Pull request merged**